.github/workflows/run-tests.yml

```
@@ -1,91 +1,95 @@
+  test:
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v5
 
-        steps:
-            - name: Checkout code
-              uses: actions/checkout@v5
+      - name: Fetch and diff against input branch
+        run: |
```

actionlint (1.7.8)

property "branch" is not defined in object type {bats-version: string; compare-branch: string; test-pattern: string; working-directory: string}

(expression)